metrics/core_metrics.md

```diff
@@ -0,0 +1,82 @@
+
+|Number| Metric                | Description    |
+|------|-----------------------|----------------|
+| 1.1  | vehicles.[status].avg                  | The average number of vehicles in [MDS status](/agency#vehicle-events). |
+| 1.2  | vehicles.[status].min                  | The minimum number of vehicles in [MDS status](/agency#vehicle-events). |
+| 1.3  | vehicles.[status].max                  | The maximum number of vehicles in [MDS status](/agency#vehicle-events). |
+| 1.4  | vehicles.[status].duration.sum         | The total duration (in seconds) vehicles spent in a specified [MDS status](/agency#vehicle-events). |
+| 1.5  | events.[event_type].count              | The number of [MDS event type](/agency#vehicle-events) received. |
+| 1.6  | trips.[start_loc/end_loc].count        | The number of trips aggregated by either the start or first enter, or end or final leave locations. |
+| 1.7  | trips.[start_loc/end_loc]_duration.avg | The average trip duration (in seconds) aggregated by either the start or first enter, or end or final leave locations. |
+| 1.8  | trips.[start_loc/end_loc]_duration.med | The median trip duration (in seconds) aggregated by either the start or first enter, or end or final leave locations. |
+| 1.9  | trips.[start_loc/end_loc]_duration.std | The standard deviation trip duration (in seconds) aggregated by either the start or first enter, or end or final leave locations. |
+| 1.10 | trips.[start_loc/end_loc]_duration.sum | The total sum of trip duration (in seconds) aggregated by either the start or first enter, or end or final leave locations. |
+| 1.11 | trips.[start_loc/end_loc]_distance.avg | The average trip distance (in meters) aggregated by either the start or first enter, or end or final leave locations. |
+| 1.12 | trips.[start_loc/end_loc]_distance.med | The median trip distance (in meters) aggregated by either the start or first enter, or end or final leave locations. |
+| 1.13 | trips.[start_loc/end_loc]_distance.std | The standard deviation trip distance (in meters) aggregated by either the start or first enter, or end or final leave locations. |
+| 1.14 | trips.[start_loc/end_loc]_distance.sum | The total sum of trip distance (in meters) aggregated by either the start or first enter, or end or final leave locations. |
+
+[Top][toc]
+
+## Dimensions
+
+The following represent the suggested MDS core metric dimensions:
+
+| Dimension    | Description |
+| ------------ | -------------------------------------------------------------------------------------------------------------------------- |
+| provider_id  | Transportation provider id issued by OMF and [tracked here](/providers.csv)                              |
+| [geography_type](/geography#geography-type)   | [MDS Geography](/geography) e.g. policy, jurisdictions, council_districts |
```

Comment:
@whereissean should this be Geography IDs, like with the Filters? I think it makes more sense than the type only, though you can select multiple geometries with one type. But it wouldn't be too much work to just use each of the IDs.

Reply:
> I like the idea but not sure they are different enough to be needed in this 1.1 version. I think adding the dockless mode and other modes later would be fine. I'd rather cut it to keep things simple, unless you have a compelling use case for dockless only metrics that I'm not seeing.

I think it is important to keep dockless metrics in.  They represent a number of the most useful metrics cities are using today to measure the utilization of dockless vehicles and what constitutes in the PRoW.  These are both pretty critical metrics for current users of MDS for dockless use cases.  They also represent a number of the metrics that create alignment with MDC metrics.